modules/oadp-checking-api-group-versions.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/advanced-topics.adoc
+
+
+:_content-type: PROCEDURE
+[id="oadp-checking-api-group-versions_{context}"]
+= Checking Kubernetes API group versions
+
+If you use Velero to backup and restore a source cluster that has a resource that has multiple Kubernetes API versions registered to it, Velero backs up only the version of that resource that uses the `preferredVersion` of that resource's Kubernetes API.
```

Comment:
@RichardHoch 
"...to back up..."
Also, this is a very cumbersome sentence. I would suggest splitting it into a few shorter ones. Something like this:
"A source cluster might have a resource with multiple Kubernetes API versions registered to it. One of those versions is the preferred version. If you use Velero to back up and restore such a source cluster, Velero backs up only the version of that resource that uses the preferred version of its Kubernetes API."